storage/pruning/fullHistoryPruningStorer_test.go

```diff
@@ -270,6 +272,7 @@ func TestFullHistoryPruningStorer_ConcurrentOperations(t *testing.T) {
 
 	startTime := time.Now()
 
+	_ = logger.SetLogLevel("*:DEBUG")
```

Comment:
delete log

Reply:
we have this on development. Let's remove-it from there

Reply:
anyway, the test is deactivated